app/models/user.rb

```diff
@@ -0,0 +1,23 @@
+class User < ApplicationRecord
+  before_save { self.email = email.downcase }
+  validates :name, presence: true, length: { maximum: 50 }, uniqueness: true
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i.freeze
+  validates :email, presence: true, length: { maximum: 255 },
+                    format: { with: VALID_EMAIL_REGEX },
+                    uniqueness: { case_sensitive: false }
+  has_many :events, foreign_key: 'creator_id'
+  has_many :attendances
```

Comment:
We are missing something here for this association to work properly. We can make this work with either a `has_many_and_belongs_to_many` as described [here](https://guides.rubyonrails.org/association_basics.html#choosing-between-has-many-through-and-has-and-belongs-to-many) or with a `has_many through` association.

I'd encourage you to use the has many through association with a join model like `EventsAtendation` with:

```
belongs_to :attendee, class_name: "User"
belongs_to :attended_event, class_name: "Event"
```
A User model will have a:
```
has_many :event_attendations
has_many :attended_events, through :event_attendations
```

And an Event model with:

```
has_many :event_attendations
has_many :attendees, through :event_attendations
```

[`REQUIRED`]

Reference: 
https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/attendee_assoc_requirement.png

Reply:
we modified the association as request. thanks for the hint